src/plugins/wb-disable/disable.js

```diff
@@ -30,15 +30,29 @@ var componentName = "wb-disable",
 			$html = wb.html,
 			i18n = wb.i18n,
 			pageUrl = wb.pageUrlParts,
-			li, param;
+			li, param,
+			i18nTemporary = {
```

Comment:
Thanks @kodecount, that's what I thought, but that doesn't help as I can only view it.